components/TripItem.js

```diff
@@ -0,0 +1,50 @@
+import { styled } from "styled-components";
+import { travel } from "@/data/data";
+
+const travelList = travel[0];
+
+export default function TripItem() {
+  return (
+    <StyledListOverview>
+      <ImageFitBox>
+        <StyledImage
+          src="/images/london.jpg"
+          alt="This is a picture of the destination city"
+        />
+      </ImageFitBox>
+      <span />
+      <ContentDetails />
```

Comment:
You can already hand over name and destination of your travellist here as props.